<?xml version="1.0"?>
<launch>
	<arg name="model" />
	<arg name="gui" default="False" />
	<param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
	<param name="use_gui" value="$(arg gui)"/>
	<node name="state_publisher_tutorials" pkg="robot1_description" type="state_publisher_tutorials" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find robot1_description)/urdf.rviz" />
</launch>
